ccdproc/tests/test_combiner.py

```diff
@@ -332,3 +332,17 @@ def test_combine_limitedmem_scale_fitsimages():
 
     np.testing.assert_array_almost_equal(avgccd.data, ccd_by_combiner.data, decimal = 4)
 
+#test the optional uncertainty function in average_combine
+def test_average_combine_uncertainty(ccd_data):
+    ccd_list = [ccd_data, ccd_data, ccd_data]
+    c = Combiner(ccd_list)
+    ccd = c.average_combine(uncertainty_func=np.sum)
+    assert np.all(ccd.uncertainty.array == np.sum(c.data_arr, 0))
```

Comment:
Just a small thing: Numpy has some testing functions: ``np.testing.assert_array_equal`` and ``np.testing.assert_array_almost_equal`` for such comparisons. See line 333 for the usage.

That's not mandatory but I think it's a bit cleaner.